studio/data/projects/project-restore-mutation.ts

```diff
@@ -11,7 +11,6 @@ export type ProjectRestoreVariables = {
 export async function restoreProject({ ref }: ProjectRestoreVariables) {
   const { data, error } = await post('/platform/projects/{ref}/restore', {
     params: { path: { ref } },
```

Comment:
Just double-checking this doesn't cause any issues - sometimes fetch without a body can cause weird issues